lib/proximal_variant.py

```diff
@@ -6,7 +6,8 @@
 import lib.utils
 
 class ProximalVariant:
-    def __init__(self, proximal_variants_vcf, pass_only):
+    #flanking_sequence_length is the number of bases (not amino acids!) to search on each side of a variant position
```

Comment:
If it's sometimes bases and sometimes amino acids that kind of makes me want it to have a different variable name for one of the cases!

Reply:
I renamed this option to `flanking_bases` for clarity.